composer.json

```diff
@@ -11,14 +11,18 @@
         }
     ],
     "require": {
-        "php": ">=5.5",
-        "dragonbe/vies": "1.0.6",
+        "php": ">=7.0",
+        "dragonbe/vies": "^1.0",
+        "escapestudios/symfony2-coding-standard": "3.0",
```

Comment:
Should packages  `escapestudios/symfony2-coding-standard`, `phpmd/phpmd`, `roave/security-advisories` and `squizlabs/php_codesniffer` not be `dev` requirements?

Reply:
yes true!